### ISSUE TITLE
v1.0.4 is missing types

### ISSUE THREAD

**Issue by camflan** [jecsham/shorthash2#10]
v1.0.3 has TS types included in the package, but v1.0.4 is missing this.

1.0.3 structure,
```
./node_modules/shorthash2
├── LICENSE
├── README.md
├── dist
|  ├── index.d.ts
|  └── index.js
├── package.json
├── src
|  └── index.ts
├── test
|  └── test.js
└── tsconfig.json

directory: 3 file: 8
```

1.0.4 structure,
```
./node_modules/shorthash2
├── LICENSE
├── README.md
├── dist
|  └── index.js
└── package.json

directory: 1 file: 4
```

**Comment by aminya** [comment]
Yes, the published package doesn't have the types
https://www.npmjs.com/package/shorthash2?activeTab=code

**Comment by camflan** [comment]
> Yes, the published package doesn't have the types https://www.npmjs.com/package/shorthash2?activeTab=code

Exactly, that's the bug 😆

**Comment by jecsham** [comment]
Fixed at v1.0.5 - #11
